pos_container_deposit/static/src/js/models.js

```diff
@@ -0,0 +1,152 @@
+/* Copyright 2024 Hunki Enterprises BV
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+odoo.define("pos_container_deposit.models", function (require) {
+    "use strict";
+
+    const {Order, Orderline} = require("point_of_sale.models");
+    const Registries = require("point_of_sale.Registries");
+    const {Gui} = require("point_of_sale.Gui");
+    const core = require("web.core");
+    const _t = core._t;
+
+    const PosDepositOrderExtension = (Order) =>
+        class PosDepositOrder extends Order {
+            add_product(product) {
+                /*
+                 * Show an error message when adding a product with a container deposit product that is not loaded
+                 */
```

Comment:
```suggestion
            /**
             * Show an error message when adding a product with a container deposit product that is not loaded
             **/
            add_product(product) {
```

nitpicking: js docstrings (other cases too)

Reply:
done